src/accessibility/AccessibilityManager.js

```diff
@@ -305,9 +305,16 @@ AccessibilityManager.prototype.addChild = function(displayObject)
 	}
 
 
+	if(displayObject.accessibleTitle) {
+		div.title = displayObject.accessibleTitle;
+	} else if (!displayObject.accessibleTitle && !displayObject.accessibleHint) {
+		div.title = 'displayObject ' + this.tabIndex;
+	}
```

Comment:
@GoodBoyDigital Sure thing

I don't know if you can add to a pull request thats been merged. Raised a
new one updating the bracket style -
https://github.com/pixijs/pixi.js/pull/2442

On Sat, Apr 2, 2016 at 7:41 PM, Mat Groves <notifications@github.com> wrote:

> In src/accessibility/AccessibilityManager.js
> <https://github.com/pixijs/pixi.js/pull/2440#discussion_r58297804>:
>
> > @@ -305,9 +305,16 @@ AccessibilityManager.prototype.addChild = function(displayObject)
> >  	}
> >
> >
> > +	if(displayObject.accessibleTitle) {
> > +		div.title = displayObject.accessibleTitle;
> > +	} else if (!displayObject.accessibleTitle && !displayObject.accessibleHint) {
> > +		div.title = 'displayObject ' + this.tabIndex;
> > +	}
>
> @andyish <https://github.com/andyish> you cool to tweak?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/pixijs/pixi.js/pull/2440/files/8470eb3eeb56ad2d7c42fa45749f67db54b83079#r58297804>
>



-- 
Kind regards,

Andrew Isherwood